packages/api-client/src/main/asset/AssetAPI.ts

```diff
@@ -30,7 +30,7 @@ export class AssetAPI {
 
   constructor(private readonly client: HttpClient) {}
 
-  getAsset(assetId: string, token?: string | null): Promise<ArrayBuffer> {
```

Comment:
This change won't work since the protobuf will call `getAsset` and there `token` can be `null`.